### ISSUE TITLE
Add rq-scheduler missing functions to rq

### ISSUE THREAD

**Issue by gabriels1234** [rq/rq#1736]
Hi!
The --with-scheduler option is a great feature. and it has helped a lot in cases where the main scheduler fails.

I use django-rq-scheduler (the new repo) that relies on rq-scheduler for a few main functionalities. It could easily fallback to vanilla rq if some of the rq-scheduler functions are ported over rq.
(such as cron(), __contains__(), etc... )

The end goal is to avoid having a separate container (extra $$$) to run the scheduler only.

Thanks!

**Comment by selwin** [comment]
Yes, that's the end goal. RQ also very recently gained a feature to track historical runs and results here so you can track the result of jobs that are run multiple times. https://github.com/rq/rq/pull/1725
